test/trento_web/views/v1/activity_log_view_test.exs

```diff
@@ -1,49 +1,11 @@
 defmodule TrentoWeb.V1.ActivityLogViewTest do
   use ExUnit.Case
 
-  import Phoenix.View
-
   import Trento.Factory
 
   alias Trento.ActivityLog.ActivityLog
   alias TrentoWeb.V1.ActivityLogView
 
-  test "should render activity_log.json" do
```

Comment:
Maybe worth keeping this test so that we also cover pagination, or we add a pagination specific test.